config/event.go

```diff
@@ -0,0 +1,19 @@
+package config
+
+import (
+	"github.com/odpf/raccoon/config/util"
+	"github.com/spf13/viper"
+)
+
+var Event event
+
+type event struct {
+	Ack int
```

Comment:
(suggestion): Instead of `int` we can define the `type` for the better readability something like
```go
type AckType int

const (
	// sends the ack, when the message is successfully receive
	Receive AckType = 0

	// sends the ack, when the message is successfully publishes to the kafka producer
	Publish AckType = 1
)

type event struct {
	Ack AckType
}
```

Reply:
+1- along the same lines, however the naming of the types could change close to the context.

```
type AckType int

const (
	// Denotes that the Acknowledgement to the consumer should be sent Asynchronously
	Asynchronous AckType = 0

	// Denotes that the Acknowledgement to the consumer should be sent Synchronously
	Synchronous AckType = 1
)

type event struct {
	Ack AckType
}
```